network-api/networkapi/templates/donate/fragments/help_page_notice.html

```diff
@@ -0,0 +1,26 @@
+{% load static i18n l10n wagtailcore_tags wagtailimages_tags %}
+
+<div>
+    <div class="tw-py-4 tw-px-0 large:tw-p-8 tw-my-24 tw-w-full tw-flex tw-flex-col large:tw-flex-row tw-items-start large:tw-items-center tw-border-t tw-border-b tw-border-black">
+        <div class="tw-flex-shrink-0 tw-grow-0 tw-basis-[50px] tw-mr-4 large:tw-mr-8">
+            <img src="{% static "_images/donate/megaphone.png" %}" alt="icon" class="tw-w-40 medium:tw-w-48 tw-h-auto" height="100" width="100">
+        </div>
+        <div>
+            <p class="tw-font-bold tw-my-6 large:tw-my-2">
+                {% blocktrans trimmed %}
+                    Our team is experiencing a backlog in donor inquiries. Please expect a delay of up to 5 business days to respond to your inquiry.
+                {% endblocktrans %}
+            </p>
+            <p class="tw-my-6 large:tw-my-2">
+                {% blocktrans trimmed %}
+                    If you are a current donor, you can use the self-service option to log into your donor portal. You can see receipts and make changes to your monthly gift. To access your log in page, refer to your link in a recent donation email.
+                {% endblocktrans %}
+            </p>
+        </div>
+    </div>
+    <p>
+        {% blocktrans trimmed %}
+            If you need help with a <a href="https://foundation.mozilla.org/?form=donate&utm_medium=web&utm_source=faq-page&utm_campaign=23-Donor-Care&utm_term=en" target="_blank">donation</a> to the Mozilla Foundation, please select the reason for your inquiry from the drop down and a donor care representative will get back to you as soon as possible.
```

Comment:
I'm wondering if the `en` in `utm_term=en` means English? If so, do we need to massage this URL a bit so the locale isn't hardcoded?

Reply:
Great question @mmmavis! I just reach out to @miaatmozilla and we decided to remove the UTM param `utm_term` for the time being now. They mentioned that its not needed

Reply:
Awesome! Thanks for double checking with stakeholders!